Makefile

```diff
@@ -91,12 +91,12 @@ bench:
 	RUST_BACKTRACE=1 cargo run --release --bin bench-tikv --features "${ENABLE_FEATURES}"
 
 pre-format:
-	# unset first in case of any previous overrides
-	rustup override unset && \
-	rustup component add rustfmt-preview
+	@# unset first in case of any previous overrides
+	@if rustup override list | grep `pwd` > /dev/null; then rustup override unset; fi
+	@rustfmt --version &>/dev/null || rustup component add rustfmt-preview
```

Comment:
rustfmt may be result in a wrong version, in this case we still need to `component add`.